static/admin/config.yml

```diff
@@ -215,6 +215,293 @@ collections:
                         },
                     ],
             }
+          - {
+                label: 'Editing Design',
+                name: 'editing_design',
+                widget: 'object',
+                required: false,
+                collapsed: true,
+                fields: [
+                        {
+                            label: 'Isoforms', # format is [GENE] NCBI Isoform(s): [entry] / [entry] / [entry]
+                            name: 'isoforms',
+                            widget: 'list',
+                            required: false,
+                            hint: '',
+                        },
+                        {
+                            label: 'cRNA Target Site',
+                            name: 'crna_target_site',
+                            widget: 'string',
+                            required: false,
+                            hint: 'Enter entire cRNA Target Site sequence. Place square brackets around PAM site. e.g. AG[TGG]ATA',
+                        },
+                        {
+                            label: 'Linker',
+                            name: 'linker',
+                            widget: 'string',
+                            required: false,
+                            hint: 'Enter entire cRNA Target Site sequence. Place square brackets around PAM site. e.g. AG[TGG]ATA',
```

Comment:
Here seems like the right spot to enter linker. We can now remove the previous entry for linker at l139-144